presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -1653,6 +1654,10 @@ public String getValidWriteIds(HiveIdentity identity, List<SchemaTableName> tabl
                         }
                     });
         }
+        catch (ConfigValSecurityException e) {
+            log.info("Could not fetch value for config " + name + " from Hive", e);
```

Comment:
The logger calls take format strings, so use that instead of concatenation. Maybe use quotes or brackets around the config name.

Should we use debug level? I’m not sure this is something we want printed every time. Otherwise, it should be warning level, since it means something is wrong that the admin might want to investigate.

The deciding factor is what the admin will do with the info. If it’s potentially useful (something they could fix), make it warning level. Otherwise hide it at debug level.

Reply:
Will change the log format.

I too was uncertain about log severity. Kept Info because it might be a surprise to admins when Hive server config does not get used in Presto if expectation is that Presto reads it from Hive.
As an action, admins can set the corresponding config in Presto but that doesnt apply in current case as we fallback to a default static value of timeout.

Reply:
Changed to debug given no admin action right now.